### PR TITLE
Fix event_callback unittest

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Even after the #67 was merged event_callback keeps
failing occasionally.

Testing shows the only invocation of the user callback
function which is guaranteed, happens after calling
drizzle_connect.

Thus the unittest has been loosed further to only check
that the call function has been invoked at least once.

### CODE REVIEW

**Comment by ben-palmer-sociomantic** [comment]
I would rephrase the commit message as:

```
Even after #67 was merged the event_callback unittest
fails occasionally.

Testing shows the only invocation of the user callback
function which is guaranteed happens after calling
drizzle_connect.

Thus the unittest has been loosened further to only check
that the call function has been invoked at least once.
```

**Comment by ben-palmer-sociomantic** [comment]
Apart from the commit message, LGTM.

**Comment by andreas-bok-sociomantic** [comment]
Updated ci msg

**Comment by ben-palmer-sociomantic** [comment]
LGTM

**Comment by ben-palmer-sociomantic** [comment]
This pull request has been rebased via `git hub pull rebase`. Original pull request HEAD was bb6420323ce5f4fe2dfab9a5a2634b47048140c6, new (rebased) HEAD is f8faa2aaf7f3dc5be6b0f47e770d9cb2715e5835
